helper/awsutil/error_test.go

```diff
@@ -0,0 +1,94 @@
+package awsutil
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	multierror "github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/vault/logical"
+)
+
+func Test_CheckAWSError(t *testing.T) {
+	testCases := []struct {
+		Name     string
+		Err      error
+		Expected error
+	}{
+		{
+			Name: "Something not checked",
+			Err:  fmt.Errorf("something"),
+		},
+		{
+			Name:     "Upstream throttle error",
+			Err:      awserr.New("Throttling", "", nil),
+			Expected: logical.ErrUpstreamRateLimited,
+		},
+		{
```

Comment:
I think this is identical to the previous case?

_squinting hard_

Reply:
Indeed! Fixed in 2fb162f9e